eras/shelley/test-suite/test/Test/Cardano/Ledger/Shelley/RulesTests.hs

```diff
@@ -132,7 +133,7 @@ testAliceSignsAlone =
 
 testAliceDoesntSign :: Assertion
 testAliceDoesntSign =
-  utxoSt' @?= Left [ScriptWitnessNotValidatingUTXOW (Set.singleton $ hashScript @C aliceOnly)]
+  utxoSt' @?= (Left . NE.singleton . ScriptWitnessNotValidatingUTXOW) wits
```

Comment:
Maybe enabling `OverloadedLists` in those test modules could be a good idea, as you did in some other places?